source/components/molecules/GroupedList/GroupedList.tsx

```diff
@@ -30,7 +34,12 @@ interface Props {
   showEditButton?: boolean;
   startEditable?: boolean;
   help?: Help;
-  children: React.ReactElement<{ category: string; editable?: boolean }>[];
+  children: React.ReactElement<{
+    category: string;
+    editable?: boolean;
+    item: SummaryListItem;
+  }>[];
+  sortCallback?: (items: SummaryListItem[]) => void;
```

Comment:
Generalize to `onCloseCallback` as discussed.

Reply:
Done